client/src/components/Indices/IndexFilter.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <span>
+        <b-input-group>
+            <DebouncedInput :delay="debounceDelay" v-slot="{ value, input }" v-model="localFilter">
+                <b-form-input
+                    :id="id"
+                    name="query"
+                    :value="value"
+                    autocomplete="off"
+                    :placeholder="placeholder | localize"
+                    data-description="filter index input"
+                    @input="input"
+                    @keyup.esc="onReset" />
+            </DebouncedInput>
+            <b-input-group-append>
+                <b-button data-description="show deleted filter toggle" @click="onHelp" title="Advanced Filtering Help">
+                    <icon icon="question" />
+                </b-button>
+            </b-input-group-append>
+        </b-input-group>
+        <b-modal v-model="showHelp" title="Filtering Options Help" ok-only>
+            <div v-html="helpHtml"></div>
+        </b-modal>
+    </span>
+</template>
+
+<script>
+import DebouncedInput from "components/DebouncedInput";
+
+import { BInputGroup, BInputGroupAppend, BButton, BModal } from "bootstrap-vue";
```

Comment:
I can see that this might be better, but is there consensus on using explicit imports? These can become pretty verbose considering our reliance on bootstrap-vue.

Reply:
I thought I read a random comment that this was preferable. If it is not I am happy to undo it. Explicit seems better than implicit to me, but I could be wrong obviously.

Reply:
Yea, I found that comment too, and yes I can see that it is better to explicitly declare those, although I am worried about the overhead but I agree with you.